qml/EntityList.qml

```diff
@@ -79,11 +79,10 @@ Rectangle {
 
                         IconSVG {
                             id: participantIcon
-                            source: "/resources/images/participant.svg"
+                            name: "participant"
```

Comment:
NIT
In general, it could be used the parameter `kind` of every `ListItem` and in `IconSVG` do a function to decide the icon (a lowercase would be enough)

Reply:
I'd leave the IconSVG agnostic of the entities or any other element of the monitor.